src/app/modules/main-module/components/main/main.component.ts

```diff
@@ -125,40 +144,36 @@ export class MainComponent implements OnInit {
     });
   }
 
-  changePositionWhenDelete() {
-    if (this.word[this.positionInput] !== '') {
+  private changePositionWhenDelete() {
+    if (this.word[this.rounds[this.contRound].positionInput] !== '') {
```

Comment:
lo mismo de antes, variable para this.word[this.rounds[this.contRound].positionInput], no se termina de entender y leer bien que pone con todo esto

Reply:
lo intentado pero no se el por que cuando lo hago en una variable se me rompe la funcionalidad

Reply:
Ya esta